### ISSUE TITLE
Unable to boot to Gnome

### ISSUE THREAD

**Issue by ErikSteiner** [anthraxx/linux-hardened#57]
I am on 5.10.7-3-MANJARO and tried to build "[linux-hardened-git 5.10.0.r968851.g1a36abfd6b77-1](https://aur.archlinux.org/packages/linux-hardened-git)" via AUR in the Software Center.
I then build "[linux-hardened-headers-git 5.10.0.r968851.g1a36abfd6b77-1](https://aur.archlinux.org/packages/linux-hardened-headers-git)" and rebooted.

Everything went find, however when I rebooted I got these messages:
```
libbpf: failed to find valid kernel BTF
libbpf: Error loading vmlinux BTF: -3
libbpf: failed to load object ''iterators_bpf'
libbpf: failed to load BPF skeleton 'iterators_bpf': -3
Failed load could be due to wrong endianness
```
I read some things and came up to this: [libbpf-tools](https://github.com/iovisor/bcc/tree/master/libbpf-tools)
I added `CONFIG_DEBUG_INFO_BTF=y` and `CONFIG_DEBUG_INFO=y` to my config in the build location. and ran `updpkgsums`. Moreover I installed `pahole 1.19`.
I then hit build in the Software Center again to rebuild `linux-hardened-git`.

After a reboot the error from above went away, however I am at a point where I only see a fast flickering underscore. TTY2 or 3 are not working as the flickering underscore blends in the login screen.

I would expect to see Gnome's login screen after reboot and be logged in automatically.

**Comment by anthraxx** [comment]
I'm not sure what your Software Center does, but a finished building creating 5.10.0.r968851.g1a36abfd6b77-1 seams about wrong and correlates with ancient bug in the vanilla kernel that was fixed around 5.10.6 of the upstream kernel: https://bugs.archlinux.org/task/69171
Try building a vanilla kernel from git matching the exact kconfig. If you have general userland or vanilla kernel issues please consult your distro or vanilla kernel.

**Comment by anthraxx** [comment]
no response for 4 weeks
